src/main/java/edu/greenblitz/robotName/commands/shooter/flyWheel/RunFlyWheelByAppropriateVelocity.java

```diff
@@ -4,9 +4,9 @@
 import edu.greenblitz.robotName.subsystems.shooter.FlyWheel.FlyWheelConstants;
 import edu.greenblitz.robotName.subsystems.swerve.Chassis.SwerveChassis;
 
-public class RunFlyWheelByApropriateVelocity extends RunFlyWheelByVelocity{
+public class RunFlyWheelByAppropriateVelocity extends RunFlyWheelByVelocity{
 
-    public RunFlyWheelByApropriateVelocity() {
+    public RunFlyWheelByAppropriateVelocity() {
```

Comment:
Consider changing to a more informative name than appropriate, same in all places